components/brave_sync/brave_sync_service_impl.cc

```diff
@@ -90,7 +90,7 @@ BraveSyncServiceImpl::BraveSyncServiceImpl(Profile* profile) :
         sync_client_.get(),
         sync_prefs_.get())),
     timer_(std::make_unique<base::RepeatingTimer>()),
-    unsynced_send_interval_(base::TimeDelta::FromMinutes(10)) {
+    unsynced_send_interval_(base::TimeDelta::FromMinutes(60)) {
```

Comment:
Why do we need to change it from 10 to 60 mins? The bug is in where we mark `sync_timestamp`.
If we bump it to 60 mins, consider there is an update for a certain bookmark, let's say url changes, it will take up to 60 mins for it to send the record.

Reply:
`unsynced_send_interval_` works here https://github.com/brave/brave-core/blob/master/components/brave_sync/client/bookmark_change_processor.cc#L943
```
      std::string last_send_time;
      node->GetMetaInfo("last_send_time", &last_send_time);
      if (!last_send_time.empty() &&
          // don't send more often than unsynced_send_interval_
          (base::Time::Now() -
              base::Time::FromJsTime(std::stod(last_send_time))) <
          unsynced_send_interval)
        continue;
```

 and affects only on re-sends.

For the regular bookmark modification we will clear `"last_send_time"`  at https://github.com/brave/brave-core/blob/master/components/brave_sync/client/bookmark_change_processor.cc#L373
```
void BookmarkChangeProcessor::BookmarkNodeChanged(BookmarkModel* model,
                                                  const BookmarkNode* node) {
  // clearing the sync_timestamp will put the record back in the `Unsynced` list
  model->DeleteNodeMetaInfo(node, "sync_timestamp");
  // also clear the last send time because this is a new change
  model->DeleteNodeMetaInfo(node, "last_send_time");

  model->SetNodeMetaInfo(node,
      "last_updated_time",
      std::to_string(base::Time::Now().ToJsTime()));
}
```
  and it will not check for `unsynced_send_interval` at https://github.com/brave/brave-core/blob/master/components/brave_sync/client/bookmark_change_processor.cc#L939 . 

So the regular send will happened once in 1 min.
And the resend will happened once in 60 min.